src/shared/data/Tile-list.es6

```diff
@@ -0,0 +1,183 @@
+          cost: 4,
+          category: 'Industrial',
+          icon: null,
+          stage: 'a',
+          immediateEffect: (player) => { player.income = player.income + 2 },
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Mint',
+          cost: 15,
+          category: 'Industrial',
+          icon: null,
+          stage: 'a',
+          immediateEffect: (player) => { player.income = player.income + 3},
+          conditionalEffect: (player) => { }
+        } 
+    ],
+
+    'b': [
+        {
+          name: 'Convenience Store',
+          cost: 6,
+          category: 'Commericial',
+          icon: null,
+          stage: 'b',
+          immediateEffect: (player) => { player.income++ },
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Mobile Home Community',
+          cost: 4,
+          category: 'Commericial',
+          icon: null,
+          stage: 'b',
+          immediateEffect: (player) => { player.population = player.population + 6 },
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Farm',
+          cost: 9,
+          category: 'Industrial',
+          icon: null,
+          stage: 'b',
+          immediateEffect: (player) => { player.reputation--},
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Landfill',
+          cost: 4,
+          category: 'Industrial',
+          icon: null,
+          stage: 'b',
+          immediateEffect: (player) => { player.income = player.income + 2 },
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Mint',
+          cost: 15,
+          category: 'Industrial',
+          icon: null,
+          stage: 'b',
+          immediateEffect: (player) => { player.income = player.income + 3},
+          conditionalEffect: (player) => { }
+        }
+    ],
+
+    'c': [
+        {
+          name: 'Convenience Store',
+          cost: 6,
+          category: 'Commericial',
+          icon: null,
+          stage: 'c',
+          immediateEffect: (player) => { player.income++ },
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Mobile Home Community',
+          cost: 4,
+          category: 'Commericial',
+          icon: null,
+          stage: 'c',
+          immediateEffect: (player) => { player.population = player.population + 6 },
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Farm',
+          cost: 9,
+          category: 'Industrial',
+          icon: null,
+          stage: 'c',
+          immediateEffect: (player) => { player.reputation--},
+          conditionalEffect: (player) => { }
+        },
+        {
+          name: 'Landfill',
+          cost: 4,
+          category: 'Industrial',
+          icon: null,
+          stage: 'c',
+          immediateEffect: (player) => { player.income = player.income + 2 },
```

Comment:
i think these immediate and conditional effects will probably need to be passed the entire gameState so that they can examine other player boards as well.  perhaps they just need to be passed all players